taca/nanopore/ONT_run_classes.py

```diff
@@ -353,6 +353,15 @@ def copy_html_report(self):
     def toulligqc_report(self):
         """Generate a QC report for the run using ToulligQC and publish it to GenStat."""
 
+        report_dir_name = "toulligqc_report"
+
+        # Do not run this function if it's output dir exists in the run dir
+        if os.path.exists(f"{self.run_abspath}/{report_dir_name}"):
```

Comment:
Would we ever want to regenerate this report?

Reply:
Possibly! If so we can just rename or remove the directory and run TACA again.